docs/developer_workflow.md

```diff
@@ -8,36 +8,26 @@ cd cluster-api-provider-nutanix
 
 ## Build source code
 <pre>
-make docker-build
-</pre>
-
-## Create test management cluster
-<pre>
-make kind-create
-</pre>
-
-## Prepare local clusterctl
-<pre>
-make prepare-local-clusterctl
+make build-snapshot
 </pre>
 
 ## Deploy cluster-api-provider-nutanix CRDs on test management cluster
 <pre>
-make deploy
+make dev.run-on-kind
```

Comment:
Does devbox have `gojq`?
```
make[1]: Leaving directory '/home/dkoshkin/src/github.com/nutanix-cloud-native/cluster-api-provider-nutanix'
bash: line 1: gojq: command not found
ERROR: image: "ko.local/cluster-api-provider-nutanix:" not present locally
make: *** [make/dev.mk:19: dev.run-on-kind] Error 1
```

Reply:
Oops! Try again, I missed it from the configuration.